src/liballoc/rc.rs

```diff
@@ -591,17 +591,11 @@ impl<T: ?Sized> Rc<T> {
     #[stable(feature = "weak_into_raw", since = "1.45.0")]
```

Comment:
I just now got very confused by this feature name, which makes it look like this is a method on `Weak`^^

Reply:
Yeah... hindsight I really wish I had suggested splitting the feature name for the `as_ptr` for `A|Rc`. Is that something we can do after-the-fact (rename these to `rc_as_ptr` or something)?

Reply:
I think you can just change it, yes... but better make it a separate PR so we can ping people to be sure.

Reply:
#73974